dataland-frontend/src/components/forms/parts/elements/derived/DriveMixFormElement.vue

```diff
@@ -0,0 +1,69 @@
+<template>
```

Comment:
Link to the modal is visible even if no data has been provided for the datapoint. In my opinion there should be no link to a model in such instances. I also believe that this would be consistent with other modals.

![image](https://github.com/d-fine/Dataland/assets/35459207/0f1bb7e5-d37d-428e-a7ac-dcdd21a72b37)

Reply:
changed

Reply:
ok